package.json

```diff
@@ -1,24 +1,22 @@
 {
   "name": "@restart/hooks",
   "version": "0.5.1",
-  "main": "lib/cjs/index.js",
-  "types": "lib/cjs/index.d.ts",
-  "module": "lib/esm/index.js",
+  "type": "module",
   "exports": {
     ".": {
-      "types": "./esm/index.d.ts",
-      "import": "./esm/index.js",
+      "types": "./lib/index.d.ts",
+      "import": "./lib/index.js",
       "require": "./cjs/index.js"
     },
     "./*": {
-      "types": "./esm/*.d.ts",
-      "import": "./esm/*.js",
+      "types": "./lib/*.d.ts",
```

Comment:
Since the types are always pointing to the ESM types, can we hoist this into a types folder and exclude the built types in the CJS folder?

Reply:
O actually this is incorrect we need to specify the types for both versions but I didn't do it here oops

Reply:
for posterity, the reason you need to do this is that the types can differ slightly (tho don't usually). Typescript will complain that it can't find types or resolve the package in some cases if you try and reuse the types for both esm/cjs in the more strict context of node/bundler resolutions